packages/ui/v2/core/Dropdown.tsx

```diff
@@ -15,7 +15,7 @@ export const DropdownMenuTrigger = forwardRef<HTMLButtonElement, DropdownMenuTri
       {...props}
       className={
         props.asChild
-          ? classNames(className, "radix-state-open:bg-gray-100 rounded-md ring-0")
+          ? classNames(className, "rounded-md ring-0 sm:focus:ring-2")
```

Comment:
i found this and I have no idea why we want bg-gray-100 here. seems like a bug. fixed

Reply:
only show ring on desktop